src/core/libmaven/mzMassSlicer.cpp

```diff
@@ -516,32 +516,55 @@ pair<bool, bool> MassSlices::_compareSlices(vector<mzSample*>& samples,
     auto highestCompIntensity = 0.0f;
     auto mzAtHighestCompIntensity = 0.0f;
     auto rtAtHighestCompIntensity = 0.0f;
-    for (auto sample : samples) {
-        // obtain EICs for the two slices
-        auto eic = sample->getEIC(mzMin,
-                                  mzMax,
-                                  rtMin,
-                                  rtMax,
-                                  1,
-                                  1,
-                                  "");
-        auto comparisonEic = sample->getEIC(comparisonMzMin,
-                                            comparisonMzMax,
-                                            comparisonRtMin,
-                                            comparisonRtMax,
-                                            1,
-                                            1,
-                                            "");
-
-        // obtain the highest intensity's mz and rt in the EICs
-        highestIntensity = eic->maxIntensity;
-        rtAtHighestIntensity = eic->rtAtMaxIntensity;
-        mzAtHighestIntensity = eic->mzAtMaxIntensity;
-        highestCompIntensity = comparisonEic->maxIntensity;
-        rtAtHighestCompIntensity = comparisonEic->rtAtMaxIntensity;
-        mzAtHighestCompIntensity = comparisonEic->mzAtMaxIntensity;
-        delete eic;
-        delete comparisonEic;
+#pragma omp parallel
```

Comment:
Good thinking with multiprocessing this block! Should help with the run time now.